.github/workflows/ci.yaml

```diff
@@ -20,7 +20,8 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Rust toolchain
-        uses: dtolnay/rust-toolchain@stable
+        uses: dtolnay/rust-toolchain@1.72.0
+        components: clippy,rstfmt
```

Comment:
This needs to be:
```
with:
  components: clippy,rustfmt
```

Reply:
```suggestion
        components: clippy,rustfmt
```

Reply:
The thing that is confusing is how this passed at all.

Reply:
> This actual "Build and Test" workflow isn't running,

ah that's confusing that syntax failures in the yaml file don't result in failed tasks?

thanks for the eyeballs: I would have actually looked if the task had had something that wasn't a check mark next to it?